ouroboros-consensus/src/Ouroboros/Consensus/Protocol/PBFT.hs

```diff
@@ -207,9 +207,10 @@ instance (PBftCrypto c, Typeable c) => OuroborosTag (PBft c) where
       case Bimap.lookupR (hashVerKey pbftIssuer) dms of
         Nothing -> throwError $ PBftNotGenesisDelegate (hashVerKey pbftIssuer) lv
         Just gk -> do
-          when (Seq.length signers >= winSize
-                && Seq.length (Seq.filter (== gk) signers) > wt)
-            $ do throwError PBftExceededSignThreshold
+          let totalSigners = Seq.length signers
+              gkSigners = Seq.length (Seq.filter (== gk) signers)
+          when (totalSigners >= winSize && gkSigners > wt)
+            $ do throwError (PBftExceededSignThreshold totalSigners gkSigners)
```

Comment:
Ah, HLint has always complained about the redundant `do` here. Now you're changing this line anyway, can you remove the `do` :slightly_smiling_face: